MobileSDKBelvedere.yml

```diff
@@ -0,0 +1,70 @@
+title: "MobileSDKBelvedere"
+packages:
+  - MobileSDKBelvedere
+  - enduser
+  - enduser_sdk
+
+parts:
+# Dialog
+  - translation:
+      key: "bsdk_belvedere_dialog_gallery"
+      title: "Label of a button that allows the user to select an image from the image gallery"
+      value: "Gallery"
+  - translation:
+      key: "bsdk_belvedere_dialog_camera"
+      title: "Label of a button that allows the user to select an image from the image gallery"
+      value: "No connection"
```

Comment:
The key, title, and value here don't seem to match. Is this correct?

Reply:
Fixed.